test/parallel/test-worker-internal-modules.mjs

```diff
@@ -0,0 +1,36 @@
+import '../common/index.mjs';
+import tmpdir from '../common/tmpdir.js';
+import assert from 'node:assert/strict';
+import { once } from 'node:events';
+import fs from 'node:fs/promises';
+import { describe, test, before } from 'node:test';
+import { Worker } from 'node:worker_threads';
+
+const accessInternalsSource = `
+import 'node:internal/freelist';
+`;
```

Comment:
Nit: is there any specific reason for this to be over multiple lines? :thinking: 

```suggestion
const accessInternalsSource = "import 'node:internal/freelist';";
```

Reply:
mh... I see, no worries, I'm fine with the multiline version :slightly_smiling_face: 

(but I'm kinda debating with myself whether it should be indented or not :thinking:)